asm/non_matchings/code_80091750/print_text1.s

```diff
@@ -121,13 +121,13 @@ glabel print_text1
 /* 094138 80093538 5120004A */  beql  $t1, $zero, .L80093664
 /* 09413C 8009353C 8E820000 */   lw    $v0, ($s4)
 .L80093540:
-/* 094140 80093540 0C024B24 */  jal   get_ascii_char_width_index
+/* 094140 80093540 0C024B24 */  jal   char_to_glyph_index
 /* 094144 80093544 02402025 */   move  $a0, $s2
 /* 094148 80093548 04400027 */  bltz  $v0, .L800935E8
 /* 09414C 8009354C 00408825 */   move  $s1, $v0
 /* 094150 80093550 8FAC0074 */  lw    $t4, 0x74($sp)
-/* 094154 80093554 3C0B800E */  lui   $t3, %hi(D_800E7E84) # $t3, 0x800e
-/* 094158 80093558 256B7E84 */  addiu $t3, %lo(D_800E7E84) # addiu $t3, $t3, 0x7e84
+/* 094154 80093554 3C0B800E */  lui   $t3, %hi(gGlyphTextureLUT) # $t3, 0x800e
```

Comment:
gLookupGlyphTexture

Reply:
or 
gGlythTextureTable

I might actually prefer that one. If you ever think your reasoning is better though, definitely make the argument.

Reply:
The choice for LUT instead of just Table was mostly due to stuff like ```main_hud_lut``` in SM64. I'm fine with changing from LUT to Table, but I can see value in mimicking SM64 in this regard.